ee/server/lib/ldap/Manager.ts

```diff
@@ -45,13 +45,13 @@ export class LDAPEEManager extends LDAPManager {
 				} else if (updateExistingUsers) {
 					await this.updateExistingUsers(ldap, converter);
 				}
+
+				converter.convertUsers({
```

Comment:
maybe it is even better removing the inner try/finally? it'd work having a single one. also easier to understand.